### ISSUE TITLE
Improve README

### ISSUE THREAD

**Issue by bradleyflood** [computerjazz/react-native-swipeable-item#56]
As a new developer to `react-native-swipable-item` I find the readme confusing and hard to follow.

I'm creating this Issue so we can track improvements to the Readme. 

My issues so far are:

1. The example is crazy complex. We need a simpler JS example, and without types. I'm a seasoned JS and TS developer myself but that demo is mind boggling.
2. The order of items is confusing too, Install and then a list of Prop Typings (which makes no sense to me as a newcomer).

Hopefully I can contribute some of these improvements myself (if I can figure out how to use the package!).

Thanks for the great work and great (looking) module.

**Comment by computerjazz** [comment]
good callouts, I really threw everything and the kitchen sink into that example didn't I?
I simplified the example (as well as the example snack) and moved the types section to the bottom.

**Comment by bradfloodx** [comment]
Nice one @computerjazz !
